src/ConnectedMode/UI/Credentials/CredentialsWnd.xaml.cs

```diff
@@ -0,0 +1,62 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Diagnostics.CodeAnalysis;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Navigation;
+using SonarLint.VisualStudio.ConnectedMode.UI.Resources;
+using SonarLint.VisualStudio.Core;
+using static SonarLint.VisualStudio.ConnectedMode.ConnectionInfo;
+
+namespace SonarLint.VisualStudio.ConnectedMode.UI.Credentials
+{
+    [ExcludeFromCodeCoverage] // UI, not really unit-testable
+    public partial class CredentialsWnd : Window
```

Comment:
```suggestion
    public partial class CredentialsDialog : Window
```
WDYT?

From https://learn.microsoft.com/en-us/windows/apps/design/controls/dialogs-and-flyouts/
```
Dialogs are modal UI overlays that provide contextual app information. Dialogs block interactions with the app window until being explicitly dismissed. They often request some kind of action from the user
```

Reply:
For now, our windows in UI folder all have the suffix "Window". But we could rename all of them. I would then propose to add this as a point to the task https://sonarsource.atlassian.net/browse/SLVS-1340

Reply:
I've created a separate task for this. https://sonarsource.atlassian.net/browse/SLVS-1343
We can probably make it a subtask of https://sonarsource.atlassian.net/browse/SLVS-1340